.chloggen/allow-pri-header-skip-syslog-stanza.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/stanza
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Replace influxdata/go-syslog with haimrubinstein/go-syslog, and add 'allow_skip_pri_header' flag to syslog setting."
```

Comment:
I would remove the mention of switching to a fork from the `note` for brevity. It's enough that it's mentioned in the `subtext`.